pytab/hwi.py

```diff
@@ -27,6 +28,28 @@
     import wmi
 
 
+def run_lshw(hardware):
+    hw_obj = subprocess.run(
```

Comment:
I changed this to `hw_json` as `hw_output` already exists.

Reply:
this is a misguided way to name things. dont name it based on what you expect from the result, because it will be confusing when "hw_json" is a "CalledProcessError"
name it something like proc, or called_proc, that indicates its a called process, then handle the returns of a CalledProcess, and then name the output hw_json if and only if you have validated content from the output of the succesful process

Reply:
Which var's did you think were unclear? I made all of them so that they would be able to be used by both so we can use the same info.

Reply:
hw_obj is a bad name for the output of subprocess.run
so is hw_json
its name should reflect that its a calledprocess. the process should be called a process, the output should be called output or something similar, and it should only be called json or obj when it IS json or an object for sure. otherwise you will wind up with var names that hide confusing error states